packages/sdk/src/doc/ds.ts

```diff
@@ -63,3 +67,6 @@ export interface ListDsMembersResponse {
   items: common.DsMember[];
   returnedRows: number /* int */;
 }
+export interface DeleteDatasetResponse {
+  success: boolean;
```

Comment:
Not a request for changes, but curious if in the future we should make `success: boolean;` a common property for all responses on the `IRpcResponse` type?